tests/t_studio.py

```diff
@@ -0,0 +1,103 @@
+import os
+import time
+import uuid
+
+import scaleapi
+
+# Testing with live key since Studio endpoints require live projects
+
+client = scaleapi.ScaleClient(os.environ["JF_STUDIO_KEY"])
+
+test_project = "vfps"
```

Comment:
Also checked PyTests in CircleCI and can confirm that it did not run through this file. So it should be formatted as a pytest file in order for each method to be validated/tested.

Reply:
Yeah, I intentionally did not want CircleCI to run this test file since I'm not 100% sure if the studio endpoints work with test api keys in mind, will confirm and fix!